plugins/ledmatrix-fantasy/manager.py

```
@@ -0,0 +1,291 @@
+
+    def _display_fallback(self) -> None:
+        """Show a fallback message when no data is available."""
+        img = self.renderer.render_no_data()
+        self.display_manager.image.paste(img, (0, 0))
+        self.display_manager.update_display()
+
+    def _advance_mode(self) -> None:
+        """Move to the next display mode, cycling through leagues.
+
+        Order: League1-Mode1 → League1-Mode2 → ... → League2-Mode1 → ...
+        Sets _cycle_complete when all leagues × all modes have been shown once.
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)